src/z3c/rml/attr.py

```diff
@@ -307,7 +317,7 @@ class File(Text):
     The value itself can eith be be a relative or absolute path. Additionally
     the following syntax is supported: [path.to.python.mpackage]/path/to/file
     """
-    open = staticmethod(urllib.urlopen)
+    open = staticmethod(six.moves.urllib.request.urlopen)
```

Comment:
There's a note about data URIs in https://docs.python.org/3/library/urllib.request.html#module-urllib.request

Not sure what the best fix is, since using URLopener directly involves setting it up. Instead of using six.moves I could just do:

    if six.PY2
       etc
    else:
       etc